build-system/tasks/presubmit-checks.js

```diff
@@ -488,17 +488,25 @@ var forbiddenTerms = {
       'build-system/amp.extern.js',
       'build-system/tasks/prepend-global/test.js',
       'build-system/tasks/prepend-global/index.js',
+      'dist.3p/current/integration.js',
```

Comment:
Is this change just alphabetization? If so, how would you feel about alphabetizing the rest of the whitelist?